easyconfigs/c/COMSOL/COMSOL-6.0.eb

```diff
@@ -0,0 +1,24 @@
+name = 'COMSOL'
+version = '6.0'
+
+homepage = 'https://www.comsol.com'
+description = """
+COMSOL Multiphysics is a general-purpose software platform, based on
+advanced numerical methods, for modeling and simulating physics-based
+problems. 
+"""
+
+toolchain = SYSTEM
+
+# source tarball has to be created manually from dvd
+# mount -o ro path-to-iso /mnt
+# cd /mnt
+# tar zcf <path-to-sources>/COMSOL-6.0.tar.gz .
+# umount /mnt
+sources = [SOURCE_TAR_GZ]
+checksums = [None]
+
+import os
+license_file=os.getenv('LMCOMSOL_LICENSE_FILE', 'license.dat')
```

Comment:
I don't think this is required? See https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/c/comsol.py#L62.

Reply:
So it should just be:

license_file=license.dat

?

Reply:
I think you could just set the environment variable in your shell and then run eb?

Reply:
Yeah, I was trying to avoid that, since it's not captured in the file.